app/models/miq_provision.rb

```diff
@@ -16,24 +16,43 @@ class MiqProvision < MiqProvisionTask
   include StateMachine
   include Tagging
 
-  alias_attribute :miq_provision_request, :miq_request   # Legacy provisioning support
-  alias_attribute :provision_type,        :request_type  # Legacy provisioning support
```

Comment:
do we want to keep these comments?

Reply:
I can add them to each method if it makes sense.

Reply:
It was one method that lost it: https://github.com/ManageIQ/manageiq/pull/23338